modyn/trainer_server/internal/grpc/trainer_server_grpc_servicer.py

```diff
@@ -222,6 +247,83 @@ def get_training_status(
         cleaned_kwargs = {k: v for k, v in response_kwargs_finished.items() if v is not None}
         return TrainingStatusResponse(**cleaned_kwargs)  # type: ignore[arg-type]
 
+    def get_values_from_queues(
+        self, training_id: int
+    ) -> Tuple[Optional[bool], Optional[int], Optional[int], Optional[int], Optional[int]]:
+        was_training = self._training_process_dict[training_id].was_training
```

Comment:
maybe add a short comment here which you had in the github thread explaining what is the idea, it's not something that is easy to understand just by looking at the code :)

Reply:
Done!